compiler/type-hint.cpp

```diff
@@ -292,6 +292,17 @@ const TypeHint *TypeHintFuture::create(PrimitiveType ptype, const TypeHint *inne
   );
 }
 
+const TypeHint *TypeHintNotNull::create(const TypeHint *inner, bool drop_not_null, bool drop_not_false) {
+  HasherOfTypeHintForOptimization hash(5192347813149011894ULL);
```

Comment:
I wish we used some hashing function here like `hash("TypeHintNotNull")` instead of magic numbers like we do it right now. But that's a story for later. :D